### DISCUSSION TITLE
Fields showing game Launcher and if "native" cloud saves (e.g Steam cloud) are already available

### DISCUSSION THREAD

**Discussion by dave-scot** [mtkennerly/ludusavi#387] [Ideas]
In the list of game saves, would it be possible, next to the "new" indicator / 3 dots / file size to include an indicator (e.g. "ST" or "EA" for steam / ea) of the launcher that the particular game uses? There might be other games like COD:MW that support cloud saves on Battle.net, but not Steam.

Related to this, would it be possible to also add an indicator of whether native cloud saves are available (enabled might be too big an ask) for that game / launcher combo? It would be handy to filter out those games (that are already being cloud-synced) so users could focus on making sure they've got all their important saves backed up

Sorry if these ideas are impossible / dumb / dupes, and thanks for your efforts so far - it's my first try of Ludusavi and it looks good!

**Comment by mtkennerly** [comment]
Hi!

Unfortunately, Ludusavi doesn't quite have enough information to tell which store a game is from. It could tell in some cases (e.g., if the save is inside of the Steam folder), but not always/consistently (e.g., if the save is in `%APPDATA%` under a game or publisher subfolder).

For games with native cloud support, there's no indicator right now, but you *can* filter them out from the "other" screen. Look for the option labelled `do not back up games with cloud support on these platforms`.

> Sorry if these ideas are impossible / dumb / dupes, and thanks for your efforts so far - it's my first try of Ludusavi and it looks good!

No worries, and thanks for your feedback! I'm glad to hear you've found it useful :)

**Reply by dave-scot** [reply]
> For games with native cloud support, there's no indicator right now, but you _can_ filter them out from the "other" screen. Look for the option labelled `do not back up games with cloud support on these platforms`.

Found it! Thanks for the heads-up
